tests/app/validation/test_date_range_validator.py

```diff
@@ -58,19 +65,69 @@ def test_date_range_valid(self):
 
         validator = DateRangeCheck()
 
-        period_from = MagicMock()
-        period_from.day.data = '01'
-        period_from.month.data = '01'
-        period_from.year.data = '2016'
+        period_from = Mock()
+        period_from.data = {
+            'day': '01',
+            'month': '01',
+            'year': '2016'
+        }
+
+        period_to = Mock()
+        period_to.data = {
+            'day': '05',
+            'month': '01',
+            'year': '2017'
+        }
+
+        mock_form = Mock()
+
+        try:
+            validator(mock_form, period_from, period_to)
+        except ValidationError:
+            self.fail('Valid date raised ValidationError')
```

Comment:
why are we not asserting the exception here?

Reply:
@jonnyshaw89 Because we are not expecting it to fail?